ci/ci/ci.py

```diff
@@ -334,7 +335,7 @@ async def update_all():
         for wb in watched_branches:
             await wb.update(request.app)
 
-    await asyncio.ensure_future(update_all())
+    request.app['task_manager'].ensure_future(update_all())
```

Comment:
was this just broken with the await before?

Reply:
`await` ing the `Task` returned by `ensure_future` has the same effect as `await update_all()`. So, broken in the sense that it hung while running the update.